api/src/main/resources/org/openmrs/module/isanteplusreports/sql/pnlsIndicatorReport/womenOnArvsScreenedForCervicalCancerAfterTreatment.sql

```diff
@@ -6,7 +6,7 @@ FROM isanteplus.patient p , isanteplus.patient_on_art pat , (SELECT p.patient_id
 	 AND A.patient_id = p.patient_id
 	AND B.patient_id = p.patient_id
 	 AND pat.date_started_cervical_cancer_status = A.second_last_status_date
-	 AND pat.cervical_cancer_status = 'NEGATIVE'
+	 AND pat.cervical_cancer_status = 'POSTIVE'
```

Comment:
This looks like a typo. Should be `POSITIVE`

Reply:
oh well , just that in the etl scripts i still named  it `POSTIVE  `https://github.com/IsantePlus/etlscript/blob/master/isanteplusreportsdmlscript.sql#L2625

Reply:
so i think you can merge this too , i will corrrect the typo in another PR at once with the etl scripts too